Automattic/wp-calypso
# Checkout: Move PrePurchaseNotices to composite-checkout directory

The `PrePurchaseNotices` component (and its dependencies) previously lived in the (unused) old checkout directory, but since that directory will soon be removed, this change moves it into the new checkout directory.

#### Testing instructions

This PR doesn't change any code, it only moves files, but we should still verify that the notices are displayed as expected.

See the testing instructions for https://github.com/Automattic/wp-calypso/pull/44943
